docs/features/software-catalog/installation.md

```diff
@@ -10,8 +10,9 @@ outlined below.
 
 ## Installing @backstage/plugin-catalog
 
-> **Note that if you used `npx @backstage/create-app`, the plugin may already be
-> present**
+> **Note that if you used `npx @backstage/create-app`, the plugin is already
+> installed and you can skip to
+> [adding entries to the catalog](#adding-entries-to-the-catalog)**
```

Comment:
These instructions should probably be relegated to an appendix somewhere, since they aren't needed with the default `create-app`. This at least makes it plainer that you _probably_ don't need to do anything here.